src/com/zebrunner/jenkins/jobdsl/factory/pipeline/TestJobFactory.groovy

```diff
@@ -54,14 +54,22 @@ public class TestJobFactory extends PipelineFactory {
             //** Triggers **//*
             def scheduling = currentSuite.getParameter("scheduling")
             if (scheduling != null && orgRepoScheduling) {
-                triggers {
-                    cron(parseSheduling(scheduling))
+                properties {
+                    pipelineTriggers {
+                        triggers {
+                            cron {
+                                spec(parseSheduling(scheduling))
+                            }
+                        }
+                    }
                 }
             }
 
             //** Properties & Parameters Area **//*
+            properties {
+                disableConcurrentBuilds()
+            }
             parameters {
-                concurrentBuild(getSuiteParameter(true, "jenkinsConcurrentBuild", currentSuite).toBoolean())
```

Comment:
this refactoring is not valid! make sure to disable only if `getSuiteParameter(true, "jenkinsConcurrentBuild", currentSuite).toBoolean()` is false!